.github/workflows/ci-actions-incremental.yml

```diff
@@ -813,12 +745,12 @@ jobs:
         uses: actions/upload-artifact@v1
         if: failure()
         with:
-          name: test-reports-native-windows-${{matrix.category}}
+          name: test-reports-native-${{matrix.category}}
```

Comment:
There is a small change here for that one windows native job:
before:
```
test-reports-native-windows-hibernate-validator
```
after:
```
test-reports-native-Windows - hibernate-validator
```
but doesn't really matter, IMHO.